app/scripts/lib/idStore.js

```diff
@@ -262,6 +262,8 @@ IdentityStore.prototype.addUnconfirmedTransaction = function (txParams, onTxDone
     query.estimateGas(txParams, function(err, result){
       if (err) return cb(err)
       txData.estimatedGas = self.addGasBuffer(result)
+      txData.txParams.gas = txData.estimatedGas
```

Comment:
@Zanibas lets remove `gas` and just use `gasLimit` (they are aliases of the same param)